.github/workflows/osff.yml

```diff
@@ -28,12 +28,12 @@ jobs:
 
     steps:
       - name: "Checkout code"
-        uses: actions/checkout@v4 # v3.1.0
+        uses: actions/checkout@v4
         with:
           persist-credentials: false
 
       - name: "Run analysis"
-        uses: ossf/scorecard-action@0864cf19026789058feabb7e87baa5f140aac736 # v2.3.1
+        uses: ossf/scorecard-action@v2
```

Comment:
This change fails --> https://github.com/cri-o/cri-o/actions/runs/7113368129/job/19365168888

Reply:
**Also, to add:**

This Pull Request was merged before I could fix some of the issues, as between me making it as "WIP", it was approved, and the bot was kind enough to merge it.

There is now https://github.com/cri-o/cri-o/pull/7560, which aims to fix some of these.

Reply:
Done.

Changed in https://github.com/cri-o/cri-o/pull/7560/commits/950f5adcf95e4b4fa599321bd6271e0e64e75fab.